scripts/suppress_yubikey_otp.pl

```diff
@@ -16,6 +16,7 @@
 
 sub event_send_text {
     my ($line, $server_rec, $wi_item_rec) = @_;
-    Irssi::signal_stop() if $line =~ /^cccccc/;
+    Irssi::signal_stop() if $line =~ /^cccc[bcdefghijklnrtuv]{40}/;
```

Comment:
Would be nice if this also matched the custom OTPs, which always start with 'vv'

Reply:
```suggestion
    Irssi::signal_stop() if $line =~ /^(cc|vv)[bcdefghijklnrtuv]{42}/;
```

Reply:
Thank you, I used today's date for the VERSION now and added your changes for the custom OTPs.